Models/User.cs

```diff
@@ -25,9 +25,6 @@ public class User
         public string UserLastName { get; set; } = null!;
```

Comment:
remove please name User from property names as we have class User, we should not duplicate class name into the name their properties